ddtrace/contrib/langchain/patch.py

```diff
@@ -774,6 +848,8 @@ def patch():
         )
         wrap("langchain", "chains.base.Chain.invoke", traced_chain_call(langchain))
         wrap("langchain", "chains.base.Chain.ainvoke", traced_chain_acall(langchain))
+        wrap("langchain_core", "runnables.base.RunnableSequence.invoke", traced_lcel_chain_invoke(langchain))
```

Comment:
This should actually probably wrap the true base class `Runnable` to capture every part of the chain (not just the initial `RunnableSequence` invocation) ... change incoming

Reply:
consequentially, `traced_lcel_chain_invoke` might change slightly, and can probably just be named `traced_lcel_chain`, or something like that